bundles.rst

```diff
@@ -6,179 +6,52 @@
 The Bundle System
 =================
 
-A bundle is similar to a plugin in other software, but even better. The key
-difference is that *everything* is a bundle in Symfony, including both the
-core framework functionality and the code written for your application.
-Bundles are first-class citizens in Symfony. This gives you the flexibility
-to use pre-built features packaged in `third-party bundles`_ or to distribute
-your own bundles. It makes it easy to pick and choose which features to enable
-in your application and to optimize them the way you want.
-
-.. note::
-
-    While you'll learn the basics here, an entire article is devoted to the
-    organization and best practices of :doc:`bundles </bundles/best_practices>`.
-
-A bundle is simply a structured set of files within a directory that implement
-a single feature. You might create a BlogBundle, a ForumBundle or
-a bundle for user management (many of these exist already as open source
-bundles). Each directory contains everything related to that feature, including
-PHP files, templates, stylesheets, JavaScript files, tests and anything else.
-Every aspect of a feature exists in a bundle and every feature lives in a
-bundle.
-
-Bundles used in your applications must be enabled by registering them in
-the ``registerBundles()`` method of the ``AppKernel`` class::
-
-    // app/AppKernel.php
-    public function registerBundles()
-    {
-        $bundles = array(
-            new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
-            new Symfony\Bundle\SecurityBundle\SecurityBundle(),
-            new Symfony\Bundle\TwigBundle\TwigBundle(),
-            new Symfony\Bundle\MonologBundle\MonologBundle(),
-            new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
-            new Symfony\Bundle\DoctrineBundle\DoctrineBundle(),
-            new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new AppBundle\AppBundle(),
-        );
-
-        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
-            $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
-            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
-            $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
-        }
-
-        return $bundles;
-    }
-
-With the ``registerBundles()`` method, you have total control over which bundles
-are used by your application (including the core Symfony bundles).
+.. caution::
+
+    In Symfony versions prior to 4.0, it was recommended to organize your own
```

Comment:
I do not really agree with this sentence. At least, it is a bit confusing as we didn't recommend to use bundles, but there only was a single AppBundle. I think that's not really clear right now.

Reply:
We'll need more opinions here, but I thought this:

* Symfony 2: organize your own code in bundles (1 bundle per feature: UserBundle, ProuctBundle, ...)
* Symfony 3: organize your own code in bundles (just 1 bundle per app: AppBundle)

Saying: *"in prior Sf versions we recommended you to organize your own code in bundles"* is compatible with that (in my opinion).

Reply:
I agree with @javiereguiluz here. "organize in bundles" doesn't have to refer to multiple bundles